src/values.rst

```diff
@@ -37,6 +48,20 @@ Two :t:`[value]s` :t:`overlap` when
 It is undefined behavior to create a :t:`value` from uninitialized memory unless
 the :t:`type` of the :t:`value` is a :t:`union type`.
 
+:dp:`fls_oqhQ62mDLckN`
+It is undefined behavior to create an :t:`allocated object` at :t:`base address`
+:c:`null`.
+
+:dp:`fls_uhwpuv6cx4ip`
+It is undefined behavior to create an :t:`allocated object` with :t:`memory
+size` ``size`` at a :t:`base address` ``base`` where ``base + size`` is greater
+than the architectures maximum :c:`usize` value.
+
+:dp:`fls_xuuFKmm181bs`
+It is undefined behavior to create an :t:`allocated object` with :t:`memory
+size` ``size`` where ``size`` is greater than the architectures maximum
+:c:`isize` value.
```

Comment:
This should be in an undefined behavior rubric, so that it shows up in the UB appendix.

Reply:
It already is (see line 45)

Reply:
Whoops!